### ISSUE TITLE
Error returned from Message API

### ISSUE THREAD

**Issue by aya-labiba** [wit-ai/wit#2749]
**Do you want to request a *feature*, report a *bug*, or ask a *question* about wit?**
question / bug

**What is the current behavior?**
The Message API is returning this error, other APIs are returning a response with the same token

{
    "code": "no-auth",
    "error": "Invalid user: FB log-in user is no longer supported"
}

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**

**What is the expected behavior?**
All other APIs are returning responses expect the Message API, we need the message API to return a success response 

**If applicable, what is the App ID where you are experiencing this issue? If you do not provide this, we cannot help.**

280962410285788

**Comment by GuilhermeXlab** [comment]
Same problem here, requested the link with the Meta account again and it still didn't resolve it.

**Comment by zjn0505** [comment]
I am having this issue since last week https://github.com/wit-ai/wit/issues/2234, getting this error my apps.
```
{
    "code": "no-auth",
    "error": "Bad auth, check token/params"
}
```

Tried regenerate server access token and it still doesn't work.

**Comment by zjn0505** [comment]
Same here. 

I can create apps, intents, utterances, with tokens in my account, but get this error when calling message API.

**Comment by ferissan** [comment]
Same here.

I have already linked my account with Meta a few years ago, and I’ve been using it successfully since then. However, recently, most of my apps are returning the following error:
```
{
    "code": "no-auth",
    "error": "Invalid user: FB log-in user is no longer supported"
}
```

I primarily create and manage my apps through the API, and this issue has not occurred before. I need assistance resolving this as it’s now affecting my core product.

**Comment by ChrisyShine** [comment]
The reason why the API query fails is because, we stop supporting the FB login in wit.ai website. We've been sending emails and Medium posts in the last several months reminding of this change. The lastest Meidum post: https://medium.com/wit-ai/wit-ai-will-delete-all-facebook-login-linked-wit-ai-accounts-on-september-1-07423765563f

Not only the API access, we will delete all the old FB login accounts in the following months. So you will not only lose the API query, but also lose all the apps linked to the old accounts.

We request users to migrate to Meta account login. And there is no self-serve way to achieve the migration. So if you want to continue use the old wit.ai apps, you need to open a new issue and provide your old account information with your new Meta account. We will help you migrate your old wit apps to your new Meta account before we start the data deletion.

**Comment by aya-labiba** [comment]
Hi @ChrisyShine,

We’ve encountered an issue where the ownership of our apps is still associated with a former employee who is no longer with the company, and unfortunately, we are unable to get in touch with them. We already have access to the apps as contributors, but we would like to know if there’s a way to transfer ownership to our accounts, which have already been migrated to Meta.

Your assistance in resolving this would be greatly appreciated.

Thank you!

**Comment by ChrisyShine** [comment]
@aya-labiba The only way to resolve the issue is to export and import the app. Since you are contributors, you should be able to export the app and then import the data as a new app. Then you will be the new app owner. 
There is no way for us to transfer the app ownership unless the request is coming from the email address who owns the wit app.

**Comment by aya-labiba** [comment]
Hi @ChrisyShine,

We have already attempted this, but unfortunately, the newly imported app is unstable and not functioning properly. We’ve raised a ticket to address the issue: [Wit.ai Issue #2751](https://github.com/wit-ai/wit/issues/2751).

Could you please let us know when the apps will be deleted, so we can plan accordingly and figure out how to resolve this? Your assistance is greatly appreciated. Thank you for your support!

**Comment by patapizza** [comment]
Hi @aya-labiba, I commented on https://github.com/wit-ai/wit/issues/2751 - it looks like this is now resolved.
